src/cluster.c

```diff
@@ -1346,6 +1361,28 @@ static void addClusterNodeReplyFromNode(RedisRaftCtx *rr,
     char *link_state = "connected";
 
     appendClusterNodeString(ret, node_id, addr, flags, master, ping_sent, pong_recv, epoch, link_state, slots);
+
+    if (self) {
+        for (unsigned int i = 0; i < sg->slot_ranges_num; i++) {
+            ShardGroupSlotRange sr = sg->slot_ranges[i];
+
+            if (sr.type == SLOTRANGE_TYPE_IMPORTING) {
+                for (unsigned int j = sr.start_slot; j <= sr.end_slot; j++) {
+                    RedisModule_Assert(j < REDIS_RAFT_HASH_SLOTS);
+                    if (migrating_map[j] != NULL && migrating_map[j]->nodes_num > 0) {
+                        appendSpecialClusterNodeString(ret, j, "<", &migrating_map[j]->nodes[0]);
+                    }
+                }
+            } else if (sr.type == SLOTRANGE_TYPE_MIGRATING) {
+                for (unsigned int j = sr.start_slot; j <= sr.end_slot; j++) {
+                    RedisModule_Assert(j < REDIS_RAFT_HASH_SLOTS);
+                    if (importing_map[j] != NULL && importing_map[j]->nodes_num > 0) {
+                        appendSpecialClusterNodeString(ret, j, ">", &importing_map[j]->nodes[0]);
+                    }
```

Comment:
Actually the behavior for these two cases is the same, the only difference is the angle bracket direction.

Reply:
I don't understand the Q?  do you mean instead of passing in the "char", we hsould pass in the type and appendSpecialClusterNodeString should handle the convesion to > or <?